tracee-ebpf/Readme.md

```diff
@@ -34,21 +34,21 @@ This will run Tracee-eBPF with no arguments, which defaults to collecting a usef
 
 You can obtain Tracee-eBPF in any of the following ways:
 1. Download from the [GitHub Releases](https://github.com/aquasecurity/tracee/releases) (`tracee.tar.gz`).
-2. Use the docker image from Docker Hub: `aquasec/tracee` (includes all the required dependencies).
+2. Use the docker image from Docker Hub: `aquasec/tracee`.
 3. Build the executable from source using `make build`. For that you will need additional development tooling.
 4. Build the executable from source in a Docker container which includes all development tooling, using `make build DOCKER=1`.
 
 ### Setup options
 
-Tracee-eBPF is made of a userspace executable (`tracee`) that drives the eBPF program, and the eBPF program itself (`tracee.bpf.$kernelversion.$traceeversion.o`). When the `tracee` is started, it will look for the eBPF program in specific places and if not found, it will attempt to build the eBPF program automatically before it starts (you can control this using the `--build-policy` flag).
+Tracee-eBPF is made of a userspace executable (`tracee-ebpf`) that drives the eBPF program, and the eBPF program itself (`tracee.bpf.$kernelversion.$traceeversion.o`). When Tracee is started, it will look for the eBPF program in specific places and if not found, it will attempt to build the eBPF program automatically before it starts (you can control this using the `--build-policy` flag).
```

Comment:
Open ended thought: I wonder if we should call Tracee (the executable) as Tracee-eBPF everywhere (except when its the rule engine) to avoid confusion.

Reply:
Good question that I also pondered about it while writing this PR. Initially I changed every single instance of tracee to tracee-ebpf, but it started to read heavily. so I changed my approach and used tracee-ebpf only when necessary and kept it tracee when possible else. My logic was that if the word relates specifically to the executable, or a property of it, then I'll change it to tracee-ebpf, but if the sentence was relevant to Tracee in general (i.e runtime security), then I keep it Tracee.
The sentence in context here is a good example: I changed the first occurrence to tracee-ebpf because it explicitly refers to the binary, but kept the second occurrence since it applies to tracee in general (regardless of how you run it).